### PR TITLE
[W33] Greg: 2015-8-10 ~ 2015-8-14

### PR DESCRIPTION
**Pull request by snowmantw**

### CODE REVIEW

**Comment by snowmantw** [comment]
r? = @timdream

File: GregWeng.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 08/10 ~ 08/14
+
+LockScreen: fix interrmitent errors on Mulet
```
Comment (timdream) [inline GregWeng.md]: @snowmantw Could you list the bug # here? Is there one?

File: GregWeng.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+* Still tracing the possible root case, and still can't successfully reproduce it on my local machine, no matter on Mac or Ubuntu
+  * As a result I can only debug it on treeheader per half hour (modifying, pushing and collecting results)
+  * Will keep trying to reproduce it on local consoles, but I have no further ideas; may ask Alexandre his best strategy
+
+Performance Tools:
+* Bug 1186762 - [Build][Raptor Transformer] Simplify the command to make Raptor + transforming process (r+)
+  * Set checkin-needed but not get landed yet. Will land it manually later.
+
+mozSettings Performance Tuning:
```
Comment (timdream) [inline GregWeng.md]: @snowmantw This is probably worthy of a bug and some progressive notes as comments too, please talk to Cervantes on that.

**Comment by timdream** [comment]
@snowmantw Please merge if you have nothing to add, or append new commit for amendments if you have things to add. Thanks.

File: GregWeng.md (hunk 2 above)
Comment (snowmantw) [inline GregWeng.md]: We don't even know what to do now, especially if I need to respect the date of this record (8/14). But today (8/18) we finally got some concrete **failures** that proven our guessing looks like wrong. I will update it on the next week report. But if you don't mind the date and a failure of guessing is valuable, I can update it here.

File: GregWeng.md (hunk 1 above)
Comment (snowmantw) [inline GregWeng.md]: Done.

**Comment by snowmantw** [comment]
@timdream I updated what I have. And please see the comments of mozSettings issues. Thanks.

**Comment by timdream** [comment]
You don't need to force push actually...

**Pull request merged**